### PR TITLE
fix: check for isEmbed

### PR DESCRIPTION
**Pull request by sean-brydon**
fixes: https://discord.com/channels/1125390093386010654/1182022772000555039/1182022772000555039

Hides "needs help" button on embed.

### CODE REVIEW

File: packages/features/bookings/Booker/components/Header.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export function Header({
 }) {
   const { t, i18n } = useLocale();
   const session = useSession();
-
+  const isEmbed = typeof window !== "undefined" && window?.isEmbed?.();
```
Comment (hariombalhara) [inline packages/features/bookings/Booker/components/Header.tsx:35]: How about using `useIsEmbed`?

**Review by hariombalhara — DISMISSED** [review]
I think we can use the hook here.

File: packages/features/bookings/Booker/components/Header.tsx (hunk 1 above)
Comment (PeerRich) [reply packages/features/bookings/Booker/components/Header.tsx:35]: yeh lets use the useIsEmbed hook her

**Comment by PeerRich** [comment]
also, the need help button only shows up in your own embed if you are the owner of the link

File: packages/features/bookings/Booker/components/Header.tsx [hunk 2]
```diff
@@ -32,7 +33,7 @@ export function Header({
...
```
Comment (PeerRich) [inline packages/features/bookings/Booker/components/Header.tsx:36]:
```suggestion
  const isEmbed = useIsEmbed();
```

Comment (PeerRich) [reply packages/features/bookings/Booker/components/Header.tsx:36]: this is better, no @hariombalhara ?

**Review by keithwillcode — APPROVED** [review]

**Review by hariombalhara — APPROVED** [review]

**Pull request merged**